core/client.go

```diff
@@ -225,10 +246,10 @@ func (c *Client) Close() error {
 
 // Wait waits client returning.
 func (c *Client) Wait() {
-	<-c.ctx.Done()
+	<-c.done
```

Comment:
`c.ctx.Done()` be used to notice goroutinue return, and The `done` channel waits connection be closed.